.github/workflows/markdown-link-checker.yml

```
@@ -0,0 +1,20 @@
+name: Check Markdown links
+
+on: [push]
+
+# on:
+#   push:
+#     branches:
+#     - master
+#   schedule:
+#   # Run everyday at 9:00 AM (See https://pubs.opengroup.org/onlinepubs/9699919799/utilities/crontab.html#tag_20_25_07)
+#   - cron: "0 9 * * *"
```

Please uncomment having the on push and on schedule both are OK to have now
```suggestion
on:
  push:
    branches:
    - master
  schedule:
  # Run everyday at 9:00 AM (See https://pubs.opengroup.org/onlinepubs/9699919799/utilities/crontab.html#tag_20_25_07)
  - cron: "0 9 * * *"
```

---

done